src/nft/components/details/AssetPriceDetails.tsx

```diff
@@ -296,8 +296,8 @@ export const NotForSale = ({ collectionName, collectionUrl }: { collectionName:
   )
 }
 
-const SubHeader = styled(ThemedText.SubHeader)`
-  color: ${({ theme }) => theme.textPrimary};
+const SubHeader = styled(ThemedText.SubHeader)<{ color: string }>`
+  color: ${({ theme, color }) => color ?? theme.textPrimary};
```

Comment:
we can just remove this and make it I think.  Pretty sure I wrote this originally and not sure why I did this lol

```<ThemedText.SubHeader>```

Reply:
that works! done thanks